server/utils/constants.go

```diff
@@ -1,6 +1,21 @@
 package utils
 
-type InterfaceName int
+import (
+	"github.com/gochain-io/gochain/v3/accounts/abi"
+)
+
+type AbiItem struct {
+	Anonymous       bool          `json:"anonymous" bson:"anonymous"`
+	Constant        bool          `json:"constant" bson:"constant"`
+	Inputs          abi.Arguments `json:"inputs" bson:"inputs"`
+	Name            string        `json:"name" bson:"name"`
+	Outputs         abi.Arguments `json:"outputs" bson:"outputs"`
+	Payable         bool          `json:"payable" bson:"payable"`
+	StateMutability string        `json:"stateMutability" bson:"stateMutability"`
+	Type            string        `json:"type" bson:"type"`
+}
+
+type InterfaceName string
```

Comment:
What about renaming this to `FunctionName`?

Reply:
@jmank88 maybe AbiDefinition like in web3?

Reply:
I personally think that `AbiDefinition` is a confusing/ambiguous name because it's not clear if it's a full ABI definition (collection of functions), or a single function. The whole web3 ecosystem seems split between `function`/`method`/`callable`, but I think that any of those make it more clear.

Reply:
I don't want to get too far off track from the PR, but `InterfaceData` could be renamed the same way, and its fields could use some short docs to clarify meaning.

Reply:
I like `FunctionName`/`FunctionData`, and I think the `Value` hash is normally called `ID`.

Reply:
@bergauz figured out?

Reply:
renamed